driver/web.go

```diff
@@ -21,7 +21,7 @@ const (
 	GET Request = "GET"
 )
 
-// Web : Driver for handling ssh executions
+// Web : Driver for handling web executions
```

Comment:
Why exactly do we have web drivers again 🙈 ?

Reply:
Lol... to help monitor web applications (we haven't fleshed out the inspectors for those but it is very valid)